assets/js/components/DateRangeSelector.js

```diff
@@ -80,9 +64,8 @@ function DateRangeSelector() {
 	const menuItems = Object.values( ranges ).map( ( range ) => range.label );
 
 	return (
-		<div className="googlesitekit-date-range-selector googlesitekit-dropdown-menu mdc-menu-surface--anchor">
+		<div ref={ menuWrapperRef } className="googlesitekit-date-range-selector googlesitekit-dropdown-menu mdc-menu-surface--anchor">
```

Comment:
No reason for two separate refs. No portals are used, both these components are children of this `div`